### ISSUE TITLE
Place for discussion with other translators

### ISSUE THREAD

**Issue by NetForce1** [AndBible/and-bible#2010]
There seem to be multiple Dutch translators active in this project. The same goes probably for other languages. Is there some common place that is used for discussion with other translators? Some texts are quite central to the app, and I would prefer to have consensus around a translation for such a text from the start, starting with a translation that we don't like and changing it later on is less then ideal.

I look around in Transifex, but could not find a place for discussion between translators of a specific language. It is possible to send messages to other Transifex members, but group messaging does not seem to be possible. GitHub issues might be a good option, but I'm not sure every translator is active here.

**Comment by tuomas2** [comment]
What would you (or anyone else following this) suggest? Telegram group (+ matrix) like we use for users group?

Github is ok but as I share your consern that not many translations follow communication here.

**Comment by NetForce1** [comment]
Several considerations that I think are important:

- The system / platform should be owned by the project / core team, and not by individual translators.
- It should support notifications (preferably via e-mail). You cannot expect people to check back every day for new messages.
- Should it be a single stream of messages, or organized around topics? Personally, I would prefer the latter.

But: I did a quick Google search (should have done that yesterday ;-) ), and it turns out I was wrong. Transifex does support discussions, they just managed to hide it well. Documentation is here: https://docs.transifex.com/teams/announcements-and-discussions/#section-discussions.

**Comment by BryceSteuer** [comment]
What platform has been used up to now for translators? Or has there been no platform whatsoever? I am curious as to how translators were communicating up to this point

**Comment by NetForce1** [comment]
Well, the Dutch are having a conversation in Transifex Discussions as we speak :smiley:

**Comment by timbze** [comment]
Yes it seems Transifex hid that feature well, I've never seen it before and had to look for some time till I found it.

IMO we can just suggest teams to use Transifex discussion then. @NetForce1 do you feel it's working ok?

**Comment by BryceSteuer** [comment]
By "AndBible translations" and "AndBible translators" do you mean that people are translating the interface text of the app? Because I think that changes based on the language setting of the Android phone. Or are the translators working to translate specific commentaries and books (modules), which there are not many of in other languages apart from English? (I saw that a team presumably had translated Commentary "Kingcomments" into Dutch. Is this the work of the translators on Transifex?

**Comment by timbze** [comment]
Yes all interface [text of the app](https://github.com/AndBible/and-bible/blob/develop/app/src/main/res/values/strings.xml), plus some more related text. E.g. JS side has another file, and Bible book names has another file

**Comment by tuomas2** [comment]
Hmm I can see only 1 message in discussions, is there more somewhere? This is what you mean by transifex discussions? https://www.transifex.com/mjdenham/teams/10122/discussions/

**Comment by timbze** [comment]
@tuomas2 you can create new discussion per language and I think you have to be member of the language to see them? At least I don't find any discussions (I'm not in Dutch team).

**Comment by tuomas2** [comment]
Hmm I added myself to dutch team and see nothing but only the one discussion (with 1 msg) I gave link to. 
![Selection_294](https://user-images.githubusercontent.com/5811789/147789629-7f51ca0b-41d9-4e04-a1c3-1a81e0f9bed2.png)

**Comment by tuomas2** [comment]
(and that I could see without being team member)

**Comment by timbze** [comment]
> (and that I could see without being team member)

I guess since you're admin/owner.

**Comment by NetForce1** [comment]
> Hmm I can see only 1 message in discussions, is there more somewhere? This is what you mean by transifex discussions? https://www.transifex.com/mjdenham/teams/10122/discussions/

Correct, that's it. If you click on the title of the post, you can see replies. I got a response from one of the other translators, so people get notified for new discussions. But I did not get a notification e-mail for the response. According to their docs notifications should be sent for all posts in a discussion you started (https://docs.transifex.com/faq/collaborating#when-do-i-receive-notifications-for-team-discussions), so it's probably a glitch.

Overall, this seems to a good enough solution. A bit Spartan maybe, but it does the job. A major plus is that it does not impose another system on translators. However, I'm basing this opinion on the experience with just one post.

**Comment by tuomas2** [comment]
What is left to do is to write instructions how to use this, put them on wiki, and advertise to translators.

**Comment by BryceSteuer** [comment]
How is it possible for me to join the AndBible organization on Transifex and possibly help out?
